Sources/Testing/Testing.docc/AddingTags.md

```diff
@@ -26,25 +26,84 @@ source files, and even test targets.
 
 ## Adding tags
 
-To add a tag to a test, use the ``Trait/tags(_:)`` trait. These traits take
-sequences of tags as arguments, and those tags are then applied to the
+To add a tag to a test, use the ``Trait/tags(_:)-505n9`` trait. These traits
+take sequences of tags as arguments, and those tags are then applied to the
 corresponding test at runtime. If they are applied to a test suite, then all
 tests in that suite inherit those tags.
 
-Tags themselves are instances of ``Tag`` and can be expressed as string literals
-or as named constants declared elsewhere:
+The testing library does not assign any semantic meaning to any tags, nor does
+the presence or absence of tags affect how the testing library runs tests.
+
+Tags themselves are instances of ``Tag`` and can be expressed as string
+literals directly in a test suite or test function's declaration. Tags can also
+be expressed as named constants declared as static members of ``Tag``. To
+declare a named constant tag, use the ``Tag()`` macro:
 
 ```swift
 extension Tag {
-  static let legallyRequired = Tag(...)
+  @Tag static var legallyRequired: Self
 }
 
-@Test("Vendor's license is valid", .tags("critical", .legallyRequired))
+@Test("Vendor's license is valid", .tags("critical"), .tags(.legallyRequired))
 func licenseValid() { ... }
 ```
 
-The testing library does not assign any semantic meaning to any tags, nor does
-the presence or absence of tags affect how the testing library runs tests.
+If two tags with the same name (`legallyRequired` in the above example) are
+declared in different files, modules, or other contexts, the testing library
+treats them as equivalent.
+
+If it is important for a tag to be distinguished from similar tags declared
+elsewhere in a package or project (or its dependencies), use
+ [reverse-DNS naming](https://en.wikipedia.org/wiki/Reverse_domain_name_notation)
+to create a unique Swift symbol name for your tag:
+
+```swift
+extension Tag {
+enum com_example_foodtruck {}
+}
+
+extension Tag.com_example_foodtruck {
+@Tag static var extraSpecial: Self
+}
+
+@Test(
+"Extra Special Sauce recipe is secret",
+.tags(.com_example_foodtruck.extraSpecial)
+)
```

Comment:
Is the indentation expected here?

Reply:
No, looks like it got messed up at some point. Will correct.